specs/badge.md

```diff
@@ -0,0 +1,80 @@
+# fast-badge
+
+## Overview
+
+*Badge* component is used to highlight an item and attract attention or flag status, such as “New”, “Sale”, or other short phrases.
```

Comment:
I'm not sure that the examples from Material or Ant fall into this overview. I do think we should consider them when we're building out this instance though. Can we expand the overview to include these?

Reply:
I assume you are talking about a badge that adorns and item that displays a count?

Reply:
correct